mubench.reviewsite/templates/detector.phtml

```diff
@@ -117,6 +117,40 @@ use MuBench\ReviewSite\Model\ReviewState;
                             <i class="fa fa-thumbs-o-down" aria-hidden="true" style="color:darkred;" title="Resolved: No"></i>
                             (<a href="<?= $review_url ?>/resolution"><i class="fa fa-handshake-o" aria-hidden="true" title="Resolution"></i></a>)
                         <?php endif; ?>
+                        <?php if($user && $misuse->hasReviewed($user)): ?>
+                            <?php
+                                $user_review = $misuse->getReview($user);
+                                $reviews = $misuse->getReviews();
+                            ?>
+                            <?php if($review_state == ReviewState::NEEDS_REVIEW && $user_review->hasPositiveHit()) : ?>
```

Comment:
i thought we also show the rank of a resolved review, even if the personal review did not identify a hit? This would not be the case if we move it up.